<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditApplicatorDialog</class>
 <widget class="QDialog" name="EditApplicatorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>229</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Applicator Info</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="pilotLabel">
       <property name="text">
        <string>Pilot:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="pilotLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="businessLabel">
       <property name="text">
        <string>Business:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="businessLineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="streetLabel">
       <property name="text">
        <string>Street:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="streetLineEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="cityLabel">
       <property name="text">
        <string>City:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="cityLineEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="stateLabel">
       <property name="text">
        <string>State:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="stateLineEdit"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="zipLabel">
       <property name="text">
        <string>ZIP:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="zipLineEdit"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="phoneLabel">
       <property name="text">
        <string>Phone:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="phoneLineEdit"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="emailLabel">
       <property name="text">
        <string>Email:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="emailLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditApplicatorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditApplicatorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
